data/human/human missions.txt

```diff
@@ -6890,3 +6890,1716 @@ mission "Hogshead's Dilemma II"
 			`As you are walking around the spaceport, you see that more than a few pilots are watching the news on nearby displays. There are mixed emotions amongst the crowd - some pilots are having a good laugh while others seem disappointed or even confused. You aren't sure what this is about until, lo and behold, the result of that Hogshead naming poll from a while ago is clearly shown on a display - Shippy McShipface. You can't say it's all that much worse than some of the names you've heard pirates give their ships before, but it still seems silly that the transport company is actually going through with the name.`
 			`	All of the hyped discussions around Hogsheads in the past gave them a brief but strong popularity amongst transport organizations looking to grab the average passenger's attention. In the end, the company responsible for this naming poll got what they wanted, and a little bit of the Hogshead's spirit has been brought back, along with the interest of several pilots. Perhaps the name "Shippy McShipface" is fitting after all.`
 				decline
+
+
+
+mission "Timothy Radrickson 1: Stowaway"
+	invisible
+	landing
+	source
+		near "Zeta Aquilae" 3 10
+		government "Republic" "Free Worlds" "Independent" "Pirate" "Neutral"
+	destination "Rand"
+	to offer
+		random < 10
+	on enter
+
+	# You will see this conversation after the first jump you make.
+	on enter
+		conversation
+			`Upon completing your jump one of your internal sensors begins to beep. You flick off the siren and check the diagnostic report; apparently there is a non-negligible heat source detected in the food storage of your ship.`
+			`	As problems go it is not a particularly alarming one; there are plenty of reasons there may be a heat spike in storage, and most of them would not require the captain of the ship to investigate.`
+			`	It is unusual, however.`
+			choice
+				`	(Investigate.)`
+					goto check
+				`	(Ignore the sensor.)`
+
+			action
+				fail
+				event "timothy radrickson: mummy" 41
+			`	It's almost certainly nothing, and you have more important things to do than chase down every little sensor glitch on your ship.`
```

Comment:
```suggestion
			`	It's almost certainly nothing, and you have more important tasks to do than chase down every little sensor glitch on your ship.`
```

Reply:
I am on the fence about this one. While tasks sounds more professional than things, "_more important things to do_" is in of itself a well used phrase, so it feels natural when I read it. Whereas "tasks to do" feels unnatural to me. Like it's being forced.

Reply:
It doesn't for me but, that's just personal preference. If you fell more about "things" than "tasks", then leave it as it is, it's already good like that.